schema.sql

```diff
@@ -43,4 +43,32 @@ ADD COLUMN owner_id int;
 
 ALTER TABLE animals
 ADD FOREIGN KEY (owner_id) REFERENCES owners(id)
-ON DELETE CASCADE;
\ No newline at end of file
+ON DELETE CASCADE;
+
+-- Create a table named vets
+CREATE TABLE vets (
+	id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+	name TEXT,
```

Comment:
- [ ] Kindly make sure that name has a varchar data type.

Reply:
Hi @sja-thedude

Please advise why.

From my research, using VARCHAR & TEXT without the max length is exactly the same, therefore there is no performance advantage. [Read More](https://www.geeksforgeeks.org/postgresql-difference-between-char-varchar-and-text/)